File: javascript/praticando/p003/bubblesort.js
// Ordenando vetor pelo bubble sort

function bubbleSort(vetor) {
    const n = vetor.length

    for (let i = 0; i < n - 1; i++) {
        for (let j = i + 1; j < n; j++) {
            if (vetor[i] > vetor[j]) {
                let aux = vetor[i]
                vetor[i] = vetor[j]
                vetor[j] = aux
            }
        }
    }

    return vetor
}


let vetorAleatorio = [56,34,2,76,432,2]
let vetorOrganizado = bubbleSort(vetorAleatorio)

console.log(`O vetor organizado ficou: ${vetorOrganizado}.`)

File: python/sortingAlgorithms/bubbleSort.py
# BUBBLE SORT

class BubbleSort:
    def sorting(self, arr):
        for i in range(len(arr) - 1):
            for j in range(i+1, len(arr)):
                if arr[i] > arr[j]:
                    aux = arr[i]
                    arr[i] = arr[j]
                    arr[j] = aux



arr = [5, 1, 7, 9, 11, 3, 2]
obj = BubbleSort()
obj.sorting(arr)
print(arr)